ExpressionEngine/Functions/Implementations/StringFunctions/ConcatFunction.cs

```diff
@@ -34,7 +34,7 @@ public override ValueTask<ValueContainer> ExecuteFunction(params ValueContainer[
             }
 
             return new ValueTask<ValueContainer>(new ValueContainer(parameters.Aggregate("",
-                (current, value) => current + AuxiliaryMethods.VcIsString(value))));
+                (current, value) => current + value)));
```

Comment:
Not needed to fix, but small performance increate using a string builder to append x strings, otherwise you will allokate so0 = s0+s1, then so1 = so0 + s2, so2 = so1  +s3 ect. 

can be optimized later (would be cool to have some performance tests/messures) fore optimizing things.

Reply:
Created an issue #79